src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -223,6 +223,84 @@ export class InstructorCoursesPageComponent implements OnInit {
     }, () => {});
   }
 
+  /**
+   * Sorts the panels of courses in order.
+   */
+  sortPanelsBy(by: string):
+      ((a: ActiveCourse, b: ActiveCourse) => number) {
+    return ((a: ActiveCourse, b: ActiveCourse): number => {
+      let strA: string;
```

Comment:
These variables name don't seem to be intuitive. Perhaps use a more relatable name?